UnitySDK/Assets/ML-Agents/Editor/Tests/RandomNormalTest.cs

```diff
@@ -1,122 +1,163 @@
-﻿using System;
+using System;
 using NUnit.Framework;
 using MLAgents.InferenceBrain;
 using MLAgents.InferenceBrain.Utils;
 
+using UnityEngine;
+using System.Collections;
+
+
+
 namespace MLAgents.Tests
 {
 
-	public class RandomNormalTest
-	{
-
-		[Test]
-		public void RandomNormalTestTwoDouble()
-		{
-			RandomNormal rn = new RandomNormal(2018);
-
-			Assert.AreEqual(-0.46666, rn.NextDouble(), 0.0001);
```

Comment:
These tests were all hardcoded to the values; I redid them to check that the mean and stddev were accounted for correctly, based on the first two values that the generator would return.